mmv1/third_party/terraform/tests/resource_dataproc_cluster_test.go.erb

```diff
@@ -690,11 +690,69 @@ func TestAccDataprocCluster_withLabels(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckDataprocClusterExists(t, "google_dataproc_cluster.with_labels", &cluster),
 
-					// We only provide one, but GCP adds three and we added goog-dataproc-autozone internally, so expect 5.
-					resource.TestCheckResourceAttr("google_dataproc_cluster.with_labels", "labels.%", "5"),
+					// We only provide one, so expect 1.
+					resource.TestCheckResourceAttr("google_dataproc_cluster.with_labels", "labels.%", "1"),
 					resource.TestCheckResourceAttr("google_dataproc_cluster.with_labels", "labels.key1", "value1"),
 				),
 			},
+			{
+				Config: testAccDataprocCluster_withLabelsUpdate(rnd),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDataprocClusterExists(t, "google_dataproc_cluster.with_labels", &cluster),
+
+					// We only provide two, so expect 2.
+					resource.TestCheckResourceAttr("google_dataproc_cluster.with_labels", "labels.%", "1"),
+					resource.TestCheckResourceAttr("google_dataproc_cluster.with_labels", "labels.key2", "value2"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccDataprocCluster_withLabelsUpdate(t *testing.T) {
```

Comment:
Can we just supersede `TestAccDataprocCluster_withLabels` with this test case by testing it between `testAccDataprocCluster_withLabels` and `testAccDataprocCluster_withoutLabels`? It's not clear what we're getting out of having both to me.

Reply:
Done.